Apps/Inc/ArrayConnection.h

```diff
@@ -8,6 +8,6 @@
 #include "Tasks.h"
 
 #include "Contactors.h"
-#include "Precharge.h"
+// #include "Precharge.h"
```

Comment:
Can we delete the commented out precharge code?

Reply:
The ArrayConnection and MotorConnection apps are getting deleted, so I commented that code out so that everything could compile. The PR that adds the new contactors app should clean this up.

Reply:
> The ArrayConnection and MotorConnection apps are getting deleted, so I commented that code out so that everything could compile. The PR that adds the new contactors app should clean this up.

For reference, that would be the rearchitected ReadSwitches app @ClarkPoon